shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/core/api/GovernanceRepositoryAPI.java

```diff
@@ -98,4 +99,11 @@
      * @param value value of data
      */
     void persist(String key, String value);
+    
+    /**
+     * renew job status.
```

Comment:
`renew`'s first character should be uppercase in javadoc, and there should be a new line between method description and params.